src/amo/components/AddonVersionCard/index.js

```diff
@@ -27,17 +27,27 @@ type Props = {|
   // An undefined version means the versions are still loading, whereas a null
   // version means that no version exists.
   version: AddonVersionType | null | void,
+  showButtonInsteadOfLink: boolean,
 |};
 
 type InternalProps = {|
   ...Props,
   versionInfo: VersionInfoType | null,
   i18n: I18nType,
   installError: string | null,
+  showButtonInsteadOfLink: boolean,
```

Comment:
This should not be in both `Props` and `InternalProps`. It only needs to be in `Props`.